cmd/istioctl/istioctl_test.go

```diff
@@ -155,3 +155,19 @@ func TestNewGet(t *testing.T) {
 		t.Fatalf("Could not delete world-cb: %v", err)
 	}
 }
+
+func TestInvalidRouteRule(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-route-rule.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
+		t.Fatalf("Did not fail when presented with invalid route rule")
+	}
+}
+
+func TestInvalidDestinationPolicy(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-destination-policy2.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
```

Comment:
I do not understand this comment.  We have a file _istio/manager/cmd/istioctl/testdata/invalid-dest-policy.yaml_ and a file _istio/manager/cmd/istioctl/testdata/invalid-dest-policy2.yaml_.  If I rename the "2" file, it will collide with the "1" file.